foundry/https_upload.py

```diff
@@ -0,0 +1,248 @@
+"""
+https_upload.py
+
+Private utility methods to upload files and/or folders to Globus using HTTPS instead of Globus Transfer.
+
+Authors:
+    Aristana Scourtas
+    Isaac Darling
+
+Last modified 12/21/22 by Isaac Darling
```

Comment:
The author list and modification dates are covered by git, and likely to fall out-of-date. Let's delete them before they are just noise.